tests/models/Entry_test.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+//TEST SETUP
+let Entry;
+let assert = chai.assert;
+let GetEntry = () =>  {
```

Comment:
Function names should be camelCased for consistency. "Classes" (technically Prototypes in JS) are what should be TitleCased.